src/zoid/buttons/component.jsx

```diff
@@ -776,9 +776,16 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
             messageType,
             offerCountryCode,
             creditProductIdentifier,
+            merchantID: serverMerchantId,
           }) => {
+            // merchantID that comes from props is an array
             const { message, buttonSessionID, currency, merchantID } = props;
 
+            // override with server id if partner does not exist
+            getLogger().addTrackingBuilder(() => ({
+              [FPTI_KEY.SELLER_ID]: merchantID?.toString() || serverMerchantId,
```

Comment:
this already exists in `sdk-client`: https://github.com/paypal/paypal-sdk-client/blob/main/src/tracking.js#L138

I would consider changing this to
```
if (serverMerchantId) {
  getLogger().addTrackingBuilder(() => ({
    [FPTI_KEY.SELLER_ID]: serverMerchantId,
  }))
}
```

Reply:
let me get this added